content/intro-to-storybook/react/it/deploy.md

```diff
@@ -5,49 +5,49 @@ description: 'Impara come deployare Storibook online'
 commit: '8652d73'
 ---
 
-Attraverso questo tutorila, abbiamo costruito componenti nell' ambiente locale della nostra macchina. Ad un certo punto, avremo bisogno di condividere il nostro lavoro per avere dei feedback dal nostro team. Procediamo quindi con il deploy di Storybook online per aiutare i nostri colleghi a rivedere l'implementazione della nostra UI.
+Attraverso questo tutorial, abbiamo costruito dei componenti nell'ambiente locale della nostra macchina. Ad un certo punto, avremo bisogno di condividere il nostro lavoro per avere dei feedback dal nostro team. Procediamo quindi con il deploy di Storybook online per aiutare i nostri colleghi a rivedere l'implementazione della nostra UI.
 
-## Exporting as a static app
+## Esportare come app statica
 
-To deploy Storybook we first need to export it as a static web app. This functionality is already built-in to Storybook and pre-configured.
+Per effettuare il deploy di Storybook per prima cosa dobbiamo esportarlo come una applicazione web statica. Questa funzionalità è già integrata in Storybook e pre-configurata.
 
-Running `yarn build-storybook` will output a static Storybook in the `storybook-static` directory, which can then be deployed to any static site hosting service.
+Eseguendo `yarn build-storybook` verrà generato uno Storybook statico nella cartella `storybook-static`, che potrà essere distribuito tramite un qualsiasi servizio di hosting per siti web statici.
 
-## Publish Storybook
+## Pubblicare Storybook
 
-This tutorial uses <a href="https://www.chromatic.com/">Chromatic</a>, a free publishing service made by the Storybook maintainers. It allows us to deploy and host our Storybook safely and securely in the cloud.
+Questo tutorial utilizza <a href="https://www.chromatic.com/">Chromatic</a>, un servizio di pubblicazione gratuito creato dai maintainers di Storybook. Chromatic ci consente di effettuare il deploy ed ospitare il nostro Storybook in modo sicuro nel cloud.
 
-### Setup a repository in GitHub
+### Creazione di un repository in GitHub
 
-Before we begin, our local code needs to sync with a remote version control service. When our project was initialized in the [Get started chapter](/intro-to-storybook/react/en/get-started/), we already initialized a local repository. At this stage we already have a set of commits that we can push to a remote one.
+Prima di iniziare, il nostro codice in locale deve essere sincronizzato con un servizio di versioning remoto. Creando il nostro progetto seguendo i passi del capitolo [Per iniziare](/intro-to-storybook/react/it/get-started/), abbiamo già inizializzato un repository locale. A questo punto abbiamo già dei commit di cui possiamo eseguire il push su di un repository remoto.
```

Comment:
_When our project was initialized in the [Get started chapter](/intro-to-storybook/react/en/get-started/), we already initialized a local repository._

Abbiamo già inizializzato il progetto quando abbiamo creato il progetto nel capitolo [Per iniziare](/intro-to-storybook/react/it/get-started/).

non sono ancora convinto nemmeno della mia versione ... può essere ancora migliorato per mantenere il senso ed essere più scorrevole

Reply:
Abbiamo già inizializzato il versionamento locale creando il progetto nel capitolo [Per iniziare]

può andare?